src/query/ts/values_test.go

```diff
@@ -1,134 +0,0 @@
-// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
Should not be deleting tests; better to replace them with tests for the new func